tools/slang-unit-test/unit-test-session-capability-not-flagged.cpp

```
@@ -0,0 +1,113 @@
+    SLANG_CHECK_ABORT(globalSession->createSession(sessionDesc, session.writeRef()) == SLANG_OK);
+
+    // Compile a minimal compute shader.  The important thing is no E36121.
+    const char* source = R"(
+[numthreads(1,1,1)]
+void main() {}
+)";
+
+    ComPtr<slang::IBlob> diagnosticBlob;
+    auto module = session->loadModuleFromSourceString(
+        "test",
+        "test.slang",
```

Cppcheck (2.21.0)

[style] The function 'fopen_s' is never used.

(unusedFunction)